src/scss/select2.scss

```diff
@@ -1,85 +1,13 @@
 .select {
-  .select-holder {
-    width: 90%;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-
-  & + .options {
-    display: none;
-  }
-
   display: flex;
```

Comment:
I beleive most of these styles aren't necessary any more.

Reply:
@tonytlwu Fixed